browser/playlist/playlist_page_handler.cc

```diff
@@ -37,6 +40,13 @@ PlaylistPageHandler::PlaylistPageHandler(
   DCHECK(profile_);
   observation_.Observe(GetPlaylistService(profile_));
 }
+#else
+PlaylistPageHandler::PlaylistPageHandler(Profile* profile) : profile_(profile) {
+  DCHECK(profile_);
+  // TODO(deeppandya) : check if we need observer for android
+  // observation_.Observe(GetPlaylistService(profile_));
```

Comment:
yeah, I also think `PlaylistPageHandler` on android also needs to observe service to notify changes to java world.